experimental/null_safety/null_safe_app/lib/goodapp.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+import 'services.dart';
+
+class GoodMyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final localizedAppName = Config.getAppName();
+    final temperatures = WeatherService.getTemperatures();
+
+    List<Widget> tempWidgets;
+    if (temperatures == null) {
+      tempWidgets = [Text('Temperature: Failed getting forecast :-(')];
+    } else {
+      // Null safety uses flow analysis. We checked for null in the branch
```

Comment:
Nit: In general, it's best to avoid first-person pronouns (I, we) in code comments. My usual cheat is to say "The app" or "this method" instead, as in "This method checked for null..."